### PR TITLE
[Kotlin Migration] CardTemplateEditor

### PR DESCRIPTION
**Pull request by BrayanDSO**
More one colossus down.

`CardTemplateEditorTest` has updates of `tempModel` nullability and `viewPager` name

Closes #10165

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateEditor.kt [hunk 1]
```diff
@@ -15,878 +15,756 @@
...
+        private fun launchCardBrowserAppearance(currentTemplate: JSONObject) {
+            val context = AnkiDroidApp.getInstance().baseContext
             if (context == null) {
-                //Catch-22, we can't notify failure as there's no context. Shouldn't happen anyway
-                Timber.w("Context was null - couldn't launch Card Browser Appearance window");
-                return;
+                // Catch-22, we can't notify failure as there's no context. Shouldn't happen anyway
+                Timber.w("Context was null - couldn't launch Card Browser Appearance window")
+                return
             }
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateEditor.kt:651]: Cleanup to remove this - fixed the underlying bug

**Review by david-allison — APPROVED** [review]
Wow! Looks great, kudos to you!!

**Comment by mikehardy** [comment]
I think temporary model just migrated, perturbing this one...CardTemplateEditor my old nemesis :laughing:

**Review by mikehardy — APPROVED** [review]
Fantastic

**Pull request merged**